locales-src/onboarding.ftl

```diff
@@ -0,0 +1,127 @@
+onboarding-tracking-protection-text2 = { -brand-short-name } helps stop websites from tracking you online, making it harder for ads to follow you around the web.
+onboarding-tracking-protection-button2 = How it Works
+
+onboarding-data-sync-title = Take Your Settings with You
+# "Sync" is short for synchronize.
+onboarding-data-sync-text2 = Sync your bookmarks, passwords, and more everywhere you use { -brand-product-name }.
+onboarding-data-sync-button2 = Sign in to { -sync-brand-short-name }
+
+onboarding-firefox-monitor-title = Stay Alert to Data Breaches
+onboarding-firefox-monitor-text = { -monitor-brand-name } monitors if your email has appeared in a data breach and alerts you if it appears in a new breach.
+onboarding-firefox-monitor-button = Sign up for Alerts
+
+onboarding-browse-privately-title = Browse Privately
+onboarding-browse-privately-text = Private Browsing clears your search and browsing history to keep it secret from anyone who uses your computer.
+onboarding-browse-privately-button = Open a Private Window
+
+onboarding-firefox-send-title = Keep Your Shared Files Private
+onboarding-firefox-send-text2 = Upload your files to { -send-brand-name } to share them with end-to-end encryption and a link that automatically expires.
+onboarding-firefox-send-button = Try { -send-brand-name }
+
+onboarding-mobile-phone-title = Get { -brand-product-name } on Your Phone
+onboarding-mobile-phone-text = Download { -brand-product-name } for iOS or Android and sync your data across devices.
+# "Mobile" is short for mobile/cellular phone, "Browser" is short for web
+# browser.
+onboarding-mobile-phone-button = Download Mobile Browser
+
+onboarding-send-tabs-title = Instantly Send Yourself Tabs
+# "Send Tabs" refers to "Send Tab to Device" feature that appears when opening a
+# tab's context menu.
+onboarding-send-tabs-text = Send Tabs instantly shares pages between your devices without having to copy, paste, or leave the browser.
+onboarding-send-tabs-button = Start Using Send Tabs
+
+onboarding-pocket-anywhere-title = Read and Listen Anywhere
+onboarding-pocket-anywhere-text2 = Save your favorite content offline with the { -pocket-brand-name } App and read, listen, and watch whenever it’s convenient for you.
+onboarding-pocket-anywhere-button = Try { -pocket-brand-name }
+
+onboarding-lockwise-passwords-title = Take Your Passwords Everywhere
+onboarding-lockwise-passwords-text2 = Keep the passwords you save secure and easily log in to your accounts with { -lockwise-brand-name }.
+onboarding-lockwise-passwords-button2 = Get the App
+
+onboarding-facebook-container-title = Set Boundaries with Facebook
+onboarding-facebook-container-text2 = { -facebook-container-brand-name } keeps your profile separate from everything else, making it harder for Facebook to target you with ads.
+onboarding-facebook-container-button = Add the Extension
+
+
+## Message strings belonging to the Return to AMO flow
```

Comment:
The onboarding.ftl is just a plain copy from mozilla-central, but I included it so it gets updated as part of exporting `locales-src/*.ftl`

Return to AMO is using the string but untouched here as migrations to onboarding.ftl (instead of newtab.ftl) will be handled in https://bugzilla.mozilla.org/show_bug.cgi?id=1556895